background/services/abilities/index.ts

```diff
@@ -113,6 +113,23 @@ export default class AbilitiesService extends BaseService<Events> {
     this.emitter.emit("newAccount", address)
   }
 
+  async removeAbilities(
+    address: HexString,
+    abilities: Ability[]
+  ): Promise<void> {
+    const cachedAbilities = await this.db.getAllAbilities()
+    const diffAbilities = cachedAbilities.filter(
+      (cachedAbility) =>
+        !abilities.find(
+          ({ abilityId }) => cachedAbility.abilityId === abilityId
+        )
+    )
```

Comment:
I doubt it will be a real problem as users probably won't own that much abilities but can we make this algorithm a little bit better by mapping `abilities` ids instead of nesting another loop?

Something like:
```suggestion
    const ids = new Set(abilities.map(a => a.abilityId))
    const diffAbilities = cachedAbilities.filter(
      (cachedAbility) =>
        !ids.has(cachedAbility.abilityId)
    )
```